gems/sorbet/bin/srb

```diff
@@ -147,4 +147,7 @@ esac
 exit $?
 # The closing comment for ruby to be ok with this file
 =end
-exec(__FILE__, *ARGV)
+
+require 'shellwords'
+
+exec(Shellwords.escape(__FILE__), *ARGV)
```

Comment:
```suggestion
exec([__FILE__, __FILE__], *ARGV)
```

The problem here is that with no arguments, `exec(__FILE__, *ARGV)` translates to just

```ruby
exec("/home/vraiment/Desktop/1. Projects/ruby_project_template/tmp/vendor/bundle/ruby/3.2.0/gems/sorbet-0.5.12388/bin/srb")
```

which is then passed to the shell for execution. Had there been one argument (e.g., `tc`), the `Kernel#exec` command would not have used the shell, and would not have split the `__FILE__` argument on any internal spaces.

The point of using `exec` here is to not depend on shell execution entirely. We can force `Kernel#exec` to avoid shell execution by passing an array as the first arg.